logger.js

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs';
+import path from 'path';
+
+const __dirname = path.dirname(new URL(import.meta.url).pathname);
+
+export default class Logger {
+  static LOG_LEVELS = {
+    INFO: 'info',
+    ERROR: 'error'
+  }
+  static #START_TIMESTAMP = new Date().toISOString();
+
+  static log = (logLevel, data) => {
+    switch (logLevel) {
+      case Logger.LOG_LEVELS.INFO:
+        console.log(data);
+        fs.appendFile(path.resolve(__dirname, `${this.#START_TIMESTAMP}.log`), `${data}\n`, (err) => this.#loggerErrorHandler(err));
+        break;
+      case Logger.LOG_LEVELS.ERROR:
+        console.error(data);
+        fs.appendFile(path.resolve(__dirname, `${this.#START_TIMESTAMP}-err.log`), `${data}\n`, (err) => this.#loggerErrorHandler(err));
+        break;
+      default:
+        console.error(`log level ${logLevel} is not supported`)
+        break;
+    }
+  }
+
+  static logSummary = (pageCount, issueCount, createdLinksCount, preexistingLinksCount, errCount, isDryRun) => {
+    const summary = `
+########################################################################################################################
+
+    ###############
+${isDryRun ? "    ##  DRY RUN  ##" : "    ###############"}
```

Comment:
super nit: Could add some indent here

Reply:
Since this is a multiline string, any added indent would actually be printed breaking the alignment of the title box.